src/ResourceManager/LocationBasedServices/LocationBasedServices.Test/ScenarioTests/LocationBasedServicesAccountTests.ps1

```diff
@@ -0,0 +1,243 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test New-AzureRmLocationBasedServicesAccount
+#>
+function Test-NewAzureRmLocationBasedServicesAccount
+{
+    # Setup
+    $rgname = Get-LocationBasedServicesManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $accountname = 'csa' + $rgname;
+        $skuname = 'S0';
+        $loc = 'West US';
```

Comment:
regions should not be hardcoded - this will keep maintainers from being able to run and record tests in some environments